### ISSUE TITLE
MongoDB

### ISSUE THREAD

**Issue by RootException** [seejohnrun/haste-server#369]
Hey,

when I insert MongoDB as the storage type, the module can not be found. Any solutions?

![a938e53b26a46fdf75da562d45fd1679](https://user-images.githubusercontent.com/46348263/110470999-dd10f000-80db-11eb-9062-048c2009d29f.png)

**Comment by likwidoxigen** [comment]
Greetings @RootException, can you post your server configuration? 
I just want to verify, did you install the mongodb npm package?
In the code I see that the source code has "mongo" as the library name yet the readme specifies that it should be "mongodb". Can you try "mongo" as the storage type and see if that works?

**Comment by RootException** [comment]
Yeah Mongo is installed. No worries, because I use it for other services.
I tried out only mongo but also...:

![dc9deadd1cdb5eabbc1ead564d9691df](https://user-images.githubusercontent.com/46348263/110685864-02d2ed80-81df-11eb-9e7d-2b59d9b9dc58.png)
![0221a7fb17a6e231126a565542f58ffe](https://user-images.githubusercontent.com/46348263/110685869-049cb100-81df-11eb-8c20-d9f50874d565.png)

**Comment by BitesizedLion** [comment]
> 
> 
> Yeah Mongo is installed. No worries, because I use it for other services.
> I tried out only mongo but also...:
> 
> ![dc9deadd1cdb5eabbc1ead564d9691df](https://user-images.githubusercontent.com/46348263/110685864-02d2ed80-81df-11eb-9e7d-2b59d9b9dc58.png)
> ![0221a7fb17a6e231126a565542f58ffe](https://user-images.githubusercontent.com/46348263/110685869-049cb100-81df-11eb-8c20-d9f50874d565.png)

Are you sure the mongodb module is specifically installed with `npm i mongodb` in the paste directory?

**Comment by johan-perso** [comment]
Try renaming lib/document_stores/mongo.js by mongodb.js

**Comment by katsumag** [comment]
Hi, I'm also having problems with MongoDB. I seem to have fixed that error by changing the storage type to "mongo" rather than "mongodb", however that results in another error, which I gather might result from the version of MongoDB I'm using being too new. However, #358 will probably address that, and I think "mongodb" as the storage type should work again

**Comment by niridya** [comment]
Is it possible to release the fix for MongoStore on master branch? It will be useful for many users.
